Week-2/exercise2.1b/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Open Animation</title>
+    <link rel="stylesheet" href="assets/style.css">
+</head>
+<body>
+    
+    <div class="hiddentext">Amit</div>
+    <div class="parent">
+        <div class="div1"></div>
+        <div class="div2"></div>
+        <div class="div3"></div>
+        <div class="div4"></div>
+    </div>
+
```

Comment:
should try this with taking everything inside a relative container and try to make text absolute to parent relative container